src/components/10-atoms/button/index.js

```diff
@@ -95,6 +95,10 @@ class AXAButton extends LitElement {
       e.stopPropagation();
       this.fakeButton.click();
     }
+
+    if (!ignoreOnClick && typeof this.onClick === 'function') {
+      this.onClick(e);
```

Comment:
You are calling `onClick` - is this the same method like in line 83 `onclick`? If so, a endless circuit could happen: Line 84 calls handleclick -> handleclick calls onclick on line 100 -> .......